src/tools/pmreorder/loggingfacility.py

```diff
@@ -90,16 +89,16 @@ def critical(self, text):
         print("CRITICAL:", text)
 
 
-def get_logger(logger_type, log_output, log_level=None):
+def get_logger(log_output, log_level=None):
     logger = None
     # check if log_level is valid
     log_level = "warning" if log_level is None else log_level
     numeric_level = getattr(logging, log_level.upper())
     if not isinstance(numeric_level, int):
         raise ValueError('Invalid log level: %s'.format(log_level.upper()))
 
-    if logger_type == "print":
+    if log_output == "print":
```

Comment:
What if the output file name is "print"? :)

Reply:
Done. Now, if  `-o` is omitted at the command line the default value will be `None` and logger print to console. Otherwise it will write to specified file.